compiler/cpp/src/thrift/generate/t_swift_generator.cc

```diff
@@ -733,10 +741,38 @@ void t_swift_generator::generate_swift_struct_implementation(ofstream& out,
   }
   if (telemetry_object_) {
     generate_swift_struct_telemetry_object_extension(out, tstruct);
+    generate_swift_struct_telemetry_event_extension(out, tstruct);
```

Comment:
this function needs to be declared in the class header. I'll push a fix shortly.

Reply:
Ah I always forget about declaring it before. You can just add it to the list of functions at the top of this file so it’s not public